app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+  before_save { self.email = email.downcase }
+
+  validates :name, presence: true, length: { maximum: 50 } 
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  validates :email, presence: true, length: { maximum: 255 },
+    format: { with: VALID_EMAIL_REGEX },
+    uniqueness: true
+    
+  has_secure_password
+  validates :password, presence: true, length: { minimum: 6 }
+
```

Comment:
try to organize the code a bit here. Group the validates callbacks together, leave some empty lines, something like:
```

 VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
 
 validates :name, presence: true, length: { maximum: 50 }
 validates :email, presence: true, length: { maximum: 255 },
                format: { with: VALID_EMAIL_REGEX },
               uniqueness: true
 validates :password, presence: true, length: { minimum: 6 }
  
 has_secure_password
```